cmds/webboot/webboot.go

```diff
@@ -0,0 +1,157 @@
+package main
```

Comment:
I'm thinking looking at the current code that if we keep everything in one file, it might get too long and unwieldy - what if we separated it into a header.go (rename to something better) which has type declarations and bookmarks, etc., and a webboot.go, which has the actual main function and logic we want? Or maybe even a separate package, not sure, @rminnich any thoughts?

Reply:
Maybe I can move the defintion of CachedDistro and DownloadedDistro  to menu pkg but indifferent .go file? Then the webboot.go just call menu pkg for everything?

Reply:
Could we do what Ron suggested and move some of this into a separate file, types.go or something?